### ISSUE TITLE
WARNING: => on line 13 isn't aligned with the previous line

### ISSUE THREAD

**Issue by mpiscaer** [rodjek/puppet-lint#37]
I have a hash in an hash,  but now the puppet-lint gives me the following error back:
WARNING: => on line 13 isn't aligned with the previous line
WARNING: => on line 14 isn't aligned with the previous line

It doesn't give me an warrning about the other line. I would say that the style gide is correct of the code.

It is effective on the following code:

``` puppet
class {'lvs::base':
    virtualeservers => {
        '192.168.2.13' => {
            vport           => '11025',
            service         => 'smtp',
            scheduler       => 'wlc',
            protocol        => 'tcp',
            checktype       => 'external',
            checkcommand    => '/path/to/checkscript',
            real_servers    => {
                'server01'   => {
                        real_server => '192.168.2.14',
                        real_port   => '25',
                        forwarding  => 'masq'
                    },
                'server02'  => {
                    real_server => '192.168.2.15',
                    real_port   => '25',
                    forwarding  => 'masq'
                }
            }
        }
    }
}
```
